framework/test/functional/mysql.test.js

```diff
@@ -86,6 +86,26 @@ describe('Test MySQL', () => {
 			expect(retrievedBlock.id).toBe(emptyBlock.id);
 		});
 
+		it('Fetch rows using whereIns', async () => {
```

Comment:
Please adjust the tests and the corresponding descriptions accordingly.

Reply:
Applied.